webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -78,7 +80,7 @@
                             <span>Extensions</span>
                         </a>
                     </li>
-                    {% if has_telemetry_environment %}
+                    {% if raw.TelemetryEnvironment %}
```

Comment:
And I hate both of those, personally. It's not a drama but I would do away with those conditional tabs too. I'd rather show a message that says there's no data than hide the tab. Consistency in navigation is better imho.